src/helpers/supportsModuleWorkers.ts

```diff
@@ -0,0 +1,18 @@
+export default function () {
+  let supports = false
+
+  const testerFunction = {
+    get type(): 'module' | 'classic' {
+      supports = true
+      return this.type
+    },
+  }
+
+  try {
+    const worker = new Worker('blob://;', testerFunction)
+    worker.terminate()
+    return supports
+  } catch (e) {
+    return supports
+  }
```

Comment:
I can't use `finally { return support }`, the linter doesn't accept this:
![image](https://user-images.githubusercontent.com/49658988/201944726-4dd0b5a6-786d-4bd7-935e-c2b73792d380.png)